src/vscodeUtils.ts

```diff
@@ -4,6 +4,18 @@ export function config(section: string, scope?: vscode.ConfigurationScope): vsco
 	return vscode.workspace.getConfiguration(section, scope);
 }
 
+export function getFolderName(folder: vscode.WorkspaceFolder): string {
+	return normalizeFolderName(folder.uri.toString());
+}
+
+// Make sure that folder uris always end with a slash
+export function normalizeFolderName(folderName: string): string {
+	if (folderName.charAt(folderName.length - 1) !== '/') {
+		folderName = folderName + '/';
```

Comment:
this maaay be the reason for the test failure, although this should be fine because the client should always work in URIs and thus `/`